org.clawsmail.claws-mail.appdata.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.clawsmail.Claws-Mail</id>
```

Comment:
The app-id should be discussed with upstream & decide which one to use. An app shouldn't have two identities.